packages/lesswrong/server/repos/UsersRepo.ts

```diff
@@ -13,4 +14,21 @@ export default class UsersRepo extends AbstractRepo<DbUser> {
       WHERE "services"->'resume'->'loginTokens' @> ('[{"hashedToken": "' || $1 || '"}]')::JSONB
     `, [hashedToken]);
   }
+  
+  getUsersWhereLocationIsInNotificationRadius(location: MongoNearLocation): Promise<Array<DbUser>> {
```

Comment:
This is an example where it would be good to write a regression test, but it would be fairly hard to write. Perhaps a point against regression tests